hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -87,7 +87,7 @@ public class HoodieClusteringConfig extends HoodieConfig {
       .key(CLUSTERING_STRATEGY_PARAM_PREFIX + "small.file.limit")
       .defaultValue(String.valueOf(300 * 1024 * 1024L))
       .sinceVersion("0.7.0")
-      .withDocumentation("Files smaller than the size specified here are candidates for clustering");
+      .withDocumentation("Files smaller than the size in bytes specified here are candidates for clustering");
```

Comment:
+1

Reply:
> +1

Here I have added the unit of file size in the description, is there any confusion?

Reply:
No. Liked the description. So +1. No action needed.